examples/atari/a3c/train_a3c.py

```diff
@@ -0,0 +1,177 @@
+from __future__ import print_function
+from __future__ import division
+from __future__ import unicode_literals
+from __future__ import absolute_import
+from builtins import *  # NOQA
+from future import standard_library
+standard_library.install_aliases()  # NOQA
+import argparse
+import os
+
+# Prevent numpy from using multiple threads
+os.environ['OMP_NUM_THREADS'] = '1'  # NOQA
+
+import chainer
+import gym
+import gym.wrappers
+import numpy as np
+
+import chainerrl
+from chainerrl.agents import a3c
+from chainerrl import experiments
+from chainerrl import links
+from chainerrl import misc
+from chainerrl.optimizers.nonbias_weight_decay import NonbiasWeightDecay
+from chainerrl.optimizers import rmsprop_async
+from chainerrl import policy
+from chainerrl import v_function
+
+from chainerrl.wrappers import atari_wrappers
+
+
+class A3CFF(chainer.ChainList, a3c.A3CModel):
```

Comment:
I'm assuming this follows the specification of the A3C paper: 

"The agents used the network architecture
from (Mnih et al., 2013). The network used a convolutional layer with 16 filters of size 8 × 8 with stride
4, followed by a convolutional layer with with 32 filters of size 4 × 4 with stride 2, followed by a fully
connected layer with 256 hidden units. All three hidden layers were followed by a rectifier nonlinearity. The
value-based methods had a single linear output unit for each action representing the action-value. The model
used by actor-critic agents had two set of outputs – a softmax output with one entry per action representing the
probability of selecting the action, and a single linear output representing the value function." (Source: A3C Paper)

Reply:
Source: Noisy Nets paper - "In each case, we used the neural network architecture from the
corresponding original papers for both the baseline and NoisyNet variant"